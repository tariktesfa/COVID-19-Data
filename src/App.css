html,
.container {
  background-color: #5687e4;
}

* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}

.display {
  display: flex;
}

.container {
  background-color: #4369b2;
}

.country {
  align-items: center;
  justify-content: space-between;
}

.search {
  flex-direction: column;
  text-align: center;
  margin-bottom: 1rem;
}

.search-input {
  margin: 1rem 8.5rem;
  padding: 0.5rem;
  background-color: white;
  outline: none;
  border-radius: 5px;
}

div.countries {
  display: flex;
  justify-content: center;
  flex-wrap: wrap;
}

a.countries {
  width: 45%;
  border: 1px solid white;
  border-radius: 2px;
  padding: 3rem 2rem 3rem 2rem;
  cursor: pointer;
  font-size: 1rem;
  color: white;
  text-decoration: none;
}

a.countries:nth-child(4n+1) {
  background-color: #5687e4;
}

a.countries:nth-child(4n+4) {
  background-color: #5687e4;
}

h2.ea-country {
  font-size: 1rem;
}

.r-country {
  justify-content: space-around;
  padding: 1rem;
  align-items: center;
  color: white;
  background-color: transparent;
}

.back-icon {
  color: white;
}

.img-world {
  width: 12rem;
}

@media screen and (min-width: 768px) {
  .img-world {
    justify-content: space-between;
    width: 30rem;
  }

  .info {
    background-color: #5687e4;
    font-size: 1.2rem;
    font-weight: bold;
    margin-right: 14rem;
  }
}

@media screen and (max-width: 969px) {
  .info {
    background-color: #5687e4;
    font-size: 1.2rem;
    font-weight: bold;
    margin-left: 1rem;
  }

  p.info {
    background-color: #5687e4;
    margin-right: 4rem;
  }
}

.info-covid {
  align-items: center;
  justify-content: space-between;
  color: white;
  background-color: #5687e4;
}

p.info {
  background-color: #5687e4;
}

.country-info {
  align-items: center;
  color: white;
  font-weight: bold;
}

li {
  display: flex;
  justify-content: space-between;
  padding: 2rem;
  color: white;
}

li:nth-child(odd) {
  background-color: #5687e4;
}

.left,
.right {
  font-size: 1.2rem;
  background-color: transparent;
}
