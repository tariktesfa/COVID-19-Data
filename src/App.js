import './App.css';

function App() {
  return (
    <div className="App" />
  );
}

export default App;
