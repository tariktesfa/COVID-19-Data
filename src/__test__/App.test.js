import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';

test('App renders correctly', () => {
  const tree = renderer.create(<App />);
  expect(tree.toJSON()).toMatchSnapshot();
});
