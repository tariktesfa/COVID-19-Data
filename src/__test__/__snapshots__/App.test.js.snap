// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="navbar"
  >
    <div
      className="nav-items"
    >
      <svg
        className="arrow"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 512 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M217.9 256L345 129c9.4-9.4 9.4-24.6 0-33.9-9.4-9.4-24.6-9.3-34 0L167 239c-9.1 9.1-9.3 23.7-.7 33.1L310.9 417c4.7 4.7 10.9 7 17 7s12.3-2.3 17-7c9.4-9.4 9.4-24.6 0-33.9L217.9 256z"
        />
      </svg>
      <h2
        className="date"
      >
        2022
      </h2>
      <h4
        className="header-text"
      >
        Countries's COVID-19 Data
      </h4>
    </div>
    <div
      className="icons"
    >
      <span
        className="icons-mic"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M16 12V6c0-2.217-1.785-4.021-3.979-4.021a.933.933 0 0 0-.209.025A4.006 4.006 0 0 0 8 6v6c0 2.206 1.794 4 4 4s4-1.794 4-4zm-6 0V6c0-1.103.897-2 2-2a.89.89 0 0 0 .163-.015C13.188 4.06 14 4.935 14 6v6c0 1.103-.897 2-2 2s-2-.897-2-2z"
          />
          <path
            d="M6 12H4c0 4.072 3.061 7.436 7 7.931V22h2v-2.069c3.939-.495 7-3.858 7-7.931h-2c0 3.309-2.691 6-6 6s-6-2.691-6-6z"
          />
        </svg>
      </span>
      <span
        className="setting"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M12 16c2.206 0 4-1.794 4-4s-1.794-4-4-4-4 1.794-4 4 1.794 4 4 4zm0-6c1.084 0 2 .916 2 2s-.916 2-2 2-2-.916-2-2 .916-2 2-2z"
          />
          <path
            d="m2.845 16.136 1 1.73c.531.917 1.809 1.261 2.73.73l.529-.306A8.1 8.1 0 0 0 9 19.402V20c0 1.103.897 2 2 2h2c1.103 0 2-.897 2-2v-.598a8.132 8.132 0 0 0 1.896-1.111l.529.306c.923.53 2.198.188 2.731-.731l.999-1.729a2.001 2.001 0 0 0-.731-2.732l-.505-.292a7.718 7.718 0 0 0 0-2.224l.505-.292a2.002 2.002 0 0 0 .731-2.732l-.999-1.729c-.531-.92-1.808-1.265-2.731-.732l-.529.306A8.1 8.1 0 0 0 15 4.598V4c0-1.103-.897-2-2-2h-2c-1.103 0-2 .897-2 2v.598a8.132 8.132 0 0 0-1.896 1.111l-.529-.306c-.924-.531-2.2-.187-2.731.732l-.999 1.729a2.001 2.001 0 0 0 .731 2.732l.505.292a7.683 7.683 0 0 0 0 2.223l-.505.292a2.003 2.003 0 0 0-.731 2.733zm3.326-2.758A5.703 5.703 0 0 1 6 12c0-.462.058-.926.17-1.378a.999.999 0 0 0-.47-1.108l-1.123-.65.998-1.729 1.145.662a.997.997 0 0 0 1.188-.142 6.071 6.071 0 0 1 2.384-1.399A1 1 0 0 0 11 5.3V4h2v1.3a1 1 0 0 0 .708.956 6.083 6.083 0 0 1 2.384 1.399.999.999 0 0 0 1.188.142l1.144-.661 1 1.729-1.124.649a1 1 0 0 0-.47 1.108c.112.452.17.916.17 1.378 0 .461-.058.925-.171 1.378a1 1 0 0 0 .471 1.108l1.123.649-.998 1.729-1.145-.661a.996.996 0 0 0-1.188.142 6.071 6.071 0 0 1-2.384 1.399A1 1 0 0 0 13 18.7l.002 1.3H11v-1.3a1 1 0 0 0-.708-.956 6.083 6.083 0 0 1-2.384-1.399.992.992 0 0 0-1.188-.141l-1.144.662-1-1.729 1.124-.651a1 1 0 0 0 .471-1.108z"
          />
        </svg>
      </span>
    </div>
  </div>
  <div
    className="hero-wrapper"
  >
    <div
      className="hero"
    >
      <div
        className="content"
      />
    </div>
  </div>
  <div
    className="search"
  >
    <p
      className="country-search"
    >
      Seach By Country: 
    </p>
    <input
      className="search-input"
      onChange={[Function]}
      placeholder="Search Country"
      type="text"
      value=""
    />
  </div>
  <div
    className="countries display"
  />
</div>
`;
